example/android/app/src/profile/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.chirp_flutter_example">
```

Comment:
Same here, the package needs to be renamed and what this file is used for? Do we need it?